server/honeytrap.go

```diff
@@ -57,10 +57,12 @@ import (
 	"github.com/honeytrap/honeytrap/pushers/eventbus"
 
 	"github.com/honeytrap/honeytrap/services"
+	_ "github.com/honeytrap/honeytrap/services/asn1-ber"
```

Comment:
Why is asn1-ber under /services? Shouldn't it be used as a normal dependency (gopkg.in/...)?

Also, why is it imported here? At a quick glance, it doesn't seem to have side effects.